packages/web/components/bridge/deposit-address-screen.tsx

```
@@ -0,0 +1,567 @@
+                      ({data.depositData.minimumDeposit.fiatValue.toString()})
+                    </>
+                  ) : null}
+                </p>
+              </SkeletonLoader>
+            </DepositInfoRow>
+            <DepositInfoRow label={<span>{t("transfer.expiresIn")}</span>}>
+              <SkeletonLoader
+                isLoaded={!isLoading}
+                className={isLoading ? "h-5 w-24" : undefined}
+              >
+                {!!expirationTimeDayjs ? (
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)